marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/cloud/aws/AWSConfiguration.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2012-2020 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.central.cloud.aws;
+
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagementClientBuilder;
+import com.amazonaws.services.simplesystemsmanagement.model.GetParameterRequest;
+import com.marklogic.hub.central.cloud.CloudProviderUtil;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.web.ServerProperties;
+import org.springframework.boot.web.embedded.tomcat.TomcatServletWebServerFactory;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+
+import java.io.InputStream;
+
+@Configuration
+@Profile("aws")
+public class AWSConfiguration {
+
+    @Value("${aws.s3.bucketName}")
+    private String bucketName;
+
+    @Value("${aws.s3.keyName}")
+    private String keyName;
+
+    @Bean
+    public ServerProperties serverProperties() {
+        return CloudProviderUtil.buildServerProperties(getKeyStorePassword());
+    }
+
+    @Bean
+    public WebServerFactoryCustomizer<TomcatServletWebServerFactory> tomcatSslStoreCustomizer() {
+        return CloudProviderUtil.configureSslStoreProvider(getKeyStorePassword(), getKeyStoreFile());
+    }
+
+    private String getKeyStorePassword() {
```

Comment:
Let's add logging to both of these private methods. Nothing special - just at the start of each method, log "Retrieving keystore password", and then at the end, "Retrieved keystore password". And call this method "retrieveKeyStorePassword" to make it clear that this is not a getter (I like for "get" to imply that it's only returning the value of a local variable). 

Same for the file - "Retrieving keystore file" and then "Retrieved keystore file". 

The two benefits are - easy confirmation that the system is using the correct profile, and we can also see how long the calls are taking too.